tests/algorithms/test_bc.py

```diff
@@ -182,8 +186,15 @@ def test_that_bc_improves_rewards(
     cartpole_venv: vec_env.VecEnv,
 ):
     # GIVEN
+    # TODO ask reviewer about preference between:
```

Comment:
2) long-term (I think @ianyfan might also work on this, so check in with me/him before working on this to avoid duplication), but probably 1) as hotfix since it may take a while to get it fixed upstream.

Reply:
Will do!